config/slskd.example.yml

```diff
@@ -203,7 +204,7 @@
 #   connection:
 #     timeout:
 #       connect: 10000
-#       inactivity: 15000
+#       inactivity: 30000
```

Comment:
btw, the timer that's backing this is reset every time a packet is sent or received over the socket, so large transfers are no problem as long as data is being sent